### ISSUE TITLE
[Bug]: Using ingress_controller = "none" generates error

### ISSUE THREAD

**Issue by mstr0m** [kube-hetzner/terraform-hcloud-kube-hetzner#1063]
### Description

Starting with version 2.8.3 using ingress_controller = "none" generates error
```
╷
│ Error: Invalid index
│ 
│   on .terraform/modules/kube-hetzner/locals.tf line 171, in locals:
│  171:   ingress_target_namespace  = var.ingress_target_namespace != "" ? var.ingress_target_namespace : local.default_ingress_namespace_mapping[var.ingress_controller]
│     ├────────────────
│     │ local.default_ingress_namespace_mapping is object with 2 attributes
│     │ var.ingress_controller is "none"
│ 
│ The given key does not identify an element in this collection value.
╵
```

2.8.5 also affected.
Feels related to [this](https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/blob/d15a9c4d131c69b97d831753788e9b6bb62a31c1/locals.tf#L167-L171).

### Kube.tf file

```terraform
module "kube-hetzner" {
  ...
  source = "kube-hetzner/kube-hetzner/hcloud"
  version = "2.8.5"
  ingress_controller = "none"
  enable_klipper_metal_lb = "true"
  ...
}
```

### Platform

Linux

**Comment by Silvest89** [comment]
https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/pull/1064

**Comment by mysticaltech** [comment]
Thanks for the fix @Silvest89. @mstr0m this should be resolved in the latest v2.8.6.
